cmds/truncate/truncate.go

```diff
@@ -36,6 +37,9 @@ func init() {
 		os.Args[0] = cmd
 		defUsage()
 	}
+
+	size = unit.MustNewUnit(unit.DefaultUnits).MustNewValue(1, unit.None)
```

Comment:
Could you gather these two and put them up in the var declaration? I don't see a need to put it in init (since it's a must anyway)

Reply:
OK